universal_transfer_operator/docs/getting-started/GETTING_STARTED.md

```diff
@@ -0,0 +1,9 @@
+# Getting Started
+
+The Universal Transfer Operator allows data transfers between any Datasets. It copies all the data from the source Dataset to the destination Dataset. The DAG author only needs to use the Universal Transfer Operator for all transfers.
+
+This ensures a consistent set of Data Providers that can read from and write to Datasets. The Universal Transfer
+Operator can use any Data Provider as a source or a destination. It also takes advantage of any existing fast and
```

Comment:
Should we add this now? since the native implementation is not added as of now?

Reply:
Yes, I think we should as that's the intent of this operator. The current state is anyways added in operators example

Reply:
Do we have this feature implemented?

Reply:
Added the specific note for this in concepts.rst